<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPrHtmlExport</class>
 <widget class="QWidget" name="KPrHtmlExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTML Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="textLabel_top">
     <property name="text">
      <string>Tick the slide should be exported :</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="KPushButton" name="kPushButton_selectAll">
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="textLabel_css">
     <property name="text">
      <string>Select the CSS template</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="KComboBox" name="kComboBox1"/>
   </item>
   <item row="2" column="2">
    <widget class="KPushButton" name="kPushButton_deselectAll">
     <property name="text">
      <string>Deselect all</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="KListWidget" name="kListBox_slides">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kPushButton_deselectAll</sender>
   <signal>clicked()</signal>
   <receiver>kListBox_slides</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kPushButton_selectAll</sender>
   <signal>clicked()</signal>
   <receiver>kListBox_slides</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
